README.md

```diff
@@ -45,19 +55,31 @@ repos:
     url: http://pkgbuild.com/~anatolik/quarry/x86_64
   sublime:
     url: https://download.sublimetext.com/arch/stable/x86_64
+download_prefetch: # optional section, add it if you want to enable prefetching
```

Comment:
It looks like you want to use crontab-like functionality for the prefetch. Do we really need it? Would a simple interval (e.g. "run prefetch every hour") work here? If not and you really want to use something more powerful then I suggest to look at cron expression parsers like this one https://github.com/gorhill/cronexpr

Reply:
Imho it is relevant cause we may want prefetching at a specific time cause network quality is also better (e.g. at night the network might be less congestionated). Didn't knew about cronexpr, I'll use it

Reply:
I have added it and tested, should be fine now.